behat/tests/bootstrap/v2/common/DebugTrait.php

```diff
@@ -77,12 +77,11 @@ public function debug($name)
         echo "$file";
     }
 
-
-    public function formatAssertResponse(
-        $expecting,
+    public function bespokeAssert(
+        $expected,
         $found,
         string $comparisonSubject,
-        string $url
+        bool $exactMatch
```

Comment:
This feels like it should be broken out to its own function rather than adding a conditional to this one to keep them focused on a single responsibility. Similar to the above comment, something like `assertStringsAreEqual()` with the same error formatting could work?

Taking that into account maybe all of this is better off in an `AssertTrait`?

Reply:
hehe ok, will go with that